python/samples/kernel-syntax-examples/memory_with_docs/main.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import PyPDF2
+import asyncio
+import semantic_kernel as sk
+import semantic_kernel.connectors.ai.open_ai as sk_oai
+from semantic_kernel.core_plugins.text_memory_plugin import TextMemoryPlugin
+from semantic_kernel.memory.semantic_text_memory import SemanticTextMemory
+from semantic_kernel.prompt_template.prompt_template_config import PromptTemplateConfig
+import os
+from dotenv import load_dotenv
+
+load_dotenv("var.env")
+api_key = os.getenv('OPENAI_API_KEY')
+
+
+collection_id = "generic"
+
+
+def extract_text_from_pdf(pdf_file_path):
+    text = ""
+    with open(pdf_file_path, "rb") as file:
+        pdf_reader = PyPDF2.PdfReader(file)
+        for page in pdf_reader.pages:
+            text += page.extract_text()
+    return text
+
+
+async def populate_memory_from_files(memory: SemanticTextMemory) -> None:
+    # Read text from .txt files and save them to the semantic memory
+    with open("data\info1.txt", "r") as file:
+        info1_text = file.read()
+    with open("data\info3.txt", "r") as file:
+        info2_text = file.read()
+    with open("data\info2.txt", "r") as file:
+        info3_text = file.read()
+    info4_text = extract_text_from_pdf("data\info4.pdf")
+
+    await memory.save_information(collection=collection_id, id="info1", text=info1_text)
+    await memory.save_information(collection=collection_id, id="info2", text=info2_text)
+    await memory.save_information(collection=collection_id, id="info3", text=info3_text)
+    await memory.save_information(collection=collection_id, id="info4", text=info4_text)
+
+
+async def setup_chat_with_memory(
```

Comment:
Our kernel examples each live in their own file. There is no need to make a sub-folder with a file `main.py`. Please follow how other kernel examples are handled.